src/modules/main/components/NewestContainer.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { Container, Grid, Button } from '@material-ui/core';
+import { IMainState } from '../store/mainReducer';
+import { IRootState } from '../../../store/rootReducer';
+import { loadNewestThunk } from '../store/actions';
+import PostPreviewCard from '../../../lib/components/PostPreview/PostPreviewCard';
+
+const NewestContainer: React.FC = () => {
+  const dispatch = useDispatch();
+  const setNewest = () => dispatch(loadNewestThunk());
+
+  const { posts: newestPosts, meta } = useSelector<
```

Comment:
```suggestion
  const { newestPosts, meta } = useSelector<
```
you can use `newestPosts` as a var name too :)

Reply:
in that case ts throws error
'Property 'newestPosts' does not exist on type 'INewestPostPayload''

Or do you mean, that I have to change the name in the interface  INewestPostPayload itself?

interface INewestPostPayload {
  posts: IPost[];<<<< 
  meta: IMeta;
}

Reply:
```suggestion
  const { posts, meta } = useSelector<
```
vice versa :)

Reply:
```suggestion
  const { posts, meta } = useSelector<
```
vice versa - you can use `posts`  as a var name

Reply:
https://github.com/ita-social-projects/dokazovi-fe/pull/39/commits/e9bd14b0817f71053bc69a73def7b8de18d1f69a

Changed to `newestPosts` in all files:)